src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -734,7 +734,7 @@ async function availability(clusterName, options) {
   let availabilityStrings = Object.entries(introducedVersions).map(
     ([os, version]) => `${os}(${version}, ${deprecatedVersions[os]})`
   );
-  return `API_DEPRECATED("${
```

Comment:
Do we need to worry about backward's compatibility here?

Reply:
In practice, not really.  I don't think anyone will be using new ZAP to regenerate the Darwin APIs on top of an old SDK revision....

Reply:
Yes, having the compatibility check only check app regen would make a lot of sense.